site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>Please use string activations.</p><p><br></p><p>Move compile out of get_model(). </p><p><br></p><p>Also, if why not use Adam() as the optimizer?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Done.
I don't know why it was not adam - @jsimsa wrote the original code.
I've changed it to "adam" now - no impact on loss.